docs/api.md

```diff
@@ -600,3 +600,132 @@ spec:
 ```
 
 See the [Operation API reference](https://redhat-appstudio.github.io/book/ref/gitops.html#operation) for details.
+
+
+## DeploymentTarget/DeploymentTargetClaim/DeploymentTargetClass
+
+See [ADR-8](https://github.com/redhat-appstudio/book/blob/main/ADR/0008-environment-provisioning.md) for details on `DeploymentTarget` (DT) / `DeploymentTargetClaim` (DTC) / `DeploymentTargetClass` (DTClass) provisioning. See [DT/DTC/DTClass step by step](deployment-target-claim-class-steps.md) for an example of how the DT/DTC/DTClass CRs operate at each step. 
+
+In short:
+- This API is patterned off of the `PersistentVolume`/`PersistentVolumeClaim`/`StorageClass` API.
+- A `DeploymentTarget` refers to a target namespace/cluster to deploy to (in the same way that a PersistentVolume represents a volume, e.g. data storage).
+- A `DeploymentTargetClaim` binds to a DeploymentTarget, becoming the owner of the DeploymentTarget (in the same way that a PersistentVolumeClaim binds to a PersistentVolume, which then allows the user to store data via the PersistentVolumeClaim)
+- A `DeploymentTargetClass` indicates the type of target to be provisioned: at present, we only support SpaceRequest provisioning, which is the ability to request a temporary Namespace on an RHTAP cluster to deploy to.
+- Finally, in order to deploy to a cluster/namespace that is bound to a DeploymentTargetClaim, you must reference that DeploymentTargetClaim from the `.spec` field of AppStudio Environment (see below for example).
+
+
+In order to deploy (using GitOps Service) to a cluster/namespace referenced by a DeploymentTargetClaim, one needs to create an AppStudio Environment resource that references that DeploymentTargetClaim, via the Enviroinment's `.spec.configuration.target.deploymentTargetClaim.claimName` field:
+```yaml
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: Environment
+metadata:
+  name: prod
+spec:
+  # (...)
+  
+  configuration:
+
+	# Target is used to reference a DeploymentTargetClaim for a target Environment.
+	# The Environment controller uses the referenced DeploymentTargetClaim to access its bounded
```

Comment:
I see a red tab in the markdown version. Removing the extra tab might fix this issue

![Screenshot from 2023-09-05 11-41-35](https://github.com/redhat-appstudio/managed-gitops/assets/21128732/50b92fbf-b7ea-4be4-801d-e8fb388410b2)